indico/modules/events/sessions/forms.py

```diff
@@ -45,6 +46,12 @@ class SessionForm(IndicoForm):
     is_poster = BooleanField(_('Poster session'), widget=SwitchWidget(),
                              description=_('Whether the session is a poster session.'))
 
+    def __init__(self, *args, **kwargs):
+        event = kwargs.pop('event')
+        super(SessionForm, self).__init__(*args, **kwargs)
+        if event.type == 'meeting':
```

Comment:
`if event.type != 'conference'` makes more sense here (in case someone accesses the page for a lecture by modifying the url; also better semantics since it's something we only support for conferences, not something we support for anything but meetings)